salesagility/SuiteCRM
# Feature/story 4487 showenginefield

Only show the engine field on search when there are more than one engine.

## Description
The engine view shows when there is only one engine, this commit will only show the select box when there are more than one engine for the user too select if there is one engine it will show a hidden field with the value.

## Motivation and Context
Too tidy up the interface for the user.

## How To Test This
1. Enable Elastic Search for the search option.
2. Check interface is working with one Engine present

![screenshot-localhost-9002-2018 11 28-14-27-40](https://user-images.githubusercontent.com/542894/49158657-9936ba00-f31a-11e8-9c47-5dcded0bf45b.png)

3. Check interface when there is more than one Engine.

![screenshot-localhost-9002-2018 11 28-14-33-22](https://user-images.githubusercontent.com/542894/49158673-9f2c9b00-f31a-11e8-9f95-795c40cb381f.png)

## Types of changes


### Final checklist
